manifests/blackbox_exporter.pp

```diff
@@ -116,12 +116,13 @@
   $options = "--config.file=${config_file} ${extra_options}"
 
   file { $config_file:
-    ensure  => file,
-    owner   => 'root',
-    group   => $group,
-    mode    => $config_mode,
-    content => template('prometheus/blackbox_exporter.yaml.erb'),
-    notify  => $notify_service,
+    ensure    => file,
+    owner     => 'root',
+    group     => $group,
+    mode      => $config_mode,
+    show_diff => false,
```

Comment:
IMO `Sensitive` support would be better. Or at least make `show_diff` configurable here. But I don't use this module, so 🤷

Reply:
Yes `Sensitive` would be better also the erb should be changed to epp